.gitignore

```diff
@@ -14,4 +14,6 @@ src/resources/version
 src/resources/sheets-api-key
 src/resources/client_secret.json
 
-.floo*
\ No newline at end of file
+.floo*
+install4j/*
```

Comment:
if you put it under build this shouldn't be necessary

Reply:
ef8587656e17d6e7fb927f12874a1b01153d9f78